README.md

```diff
@@ -215,7 +220,8 @@ SELECT * FROM table WHERE col_double = cast(? AS DOUBLE) OR col_timestamp = CAST
 ### Response rows
 
 When reading response rows, the driver supports most Trino data types, except:
-* time and timestamps with precision - all time types are returned as `time.Time`
+* time and timestamps with precision - all time types are returned as `time.Time`.
+  All precisions up to nanoseconds (`TIMESTAMP(9)` or `TIME(9)`) are supported.  
```

Comment:
That can be a bit out of the scope of this PR, since what I requested should go into the description of `trino.NullTime`, in the [Response rows](https://github.com/trinodb/trino-go-client#response-rows) section, but there's not much there now. This should not block this PR from getting merged.